allensdk/brain_observatory/behavior/session_apis/data_io/ophys_lims_api.py

```diff
@@ -401,18 +401,6 @@ def get_foraging_id(self) -> str:
                 """.format(self.get_ophys_experiment_id())
         return self.lims_db.fetchone(query, strict=True)
 
-    @memoize
-    def get_rig_name(self) -> str:
```

Comment:
I think you meant to rename this to `get_equipment_name()` and not delete the entire function?

Reply:
@njmei I made the mistake of thinking the behavior ophys session equipment and the associated behavior session equipment would be the same, but it looks like it is not always the case. I will add it back in.

Reply:
That scares me if you are finding cases where those equipment values are different.  If I scour the LIMS database, I only see four mismatches and they are all for test sessions run by MPE.  Are you able to provide examples of the mismatches that you have seen?

Reply:
@wbwakeman  I also see cases where the behavior session equipment is null and the ophys session equipment is not null or vice versa. Attaching the output of the query.
[data-1614789104869.txt](https://github.com/AllenInstitute/AllenSDK/files/6077528/data-1614789104869.txt)

Reply:
@wbwakeman just confirming whether I was correct to think this is duplicate code and that we can rely on the `behavior_session.equipment_id` for the equipment or whether we need to differentiate between behavior_session equipment and ophys_session equipment.

Reply:
All of the sessions in data-1614789104869.txt are real junk and definitely safe to ignore.

I think it is okay to rely on `behavior_session.equipment_id`